README.md

```diff
@@ -100,42 +100,54 @@ If `buildKey` and `buildName` parameters are not provided, a standard name will
     bitbucketStatusNotify(
       buildState: 'INPROGRESS',
       buildKey: 'build',
-      buildName: 'Build'
+      buildName: 'Build',
+      repoSlug: 'my-awesome-project',
```

Comment:
Why 'repoSlug' instead of just 'repository'?

Reply:
The rest of the module has several different references to `repoUrl` and `repoUri`, calling the slug for the repo `repoSlug` felt coherent to me. No problem on my side in renaming it to `repository` anyway :)

Reply:
I'm just a wandering user who happened to find this pull request, and thought: "Hey this is nice! I need this!", so feel free to ignore my comments.
The reason I like 'repository' better is because it is part of the public API (of the 'bitbucketStatusNotify' pipeline command). A user will understand right away what 'repository' means, and how it can be used, 'repoSlug' sounds less clear.

Reply:
You're right, but I also wanted to make clear that users need to pass only the slug of the repo and not the full URL in here.

Actually, some improvements could be made to detect if the parameter value is a complete URL of just a slug and act properly.

Reply:
Aha, I understand. In that case 'repositoryName' would be nice.